### PR TITLE
Content: Getting Started with Node-RED Dashboard 2.0

### LINKED ISSUE #1706: Blog: Getting Started with Node-RED Dashboard 2.0
### Description

_No response_

### PR DESCRIPTION
**Pull request by joepavitt**
## Description

- Article Written

Due for publication on 27th March, 2024

## Related Issue(s)

Closes #1706 https://github.com/FlowFuse/customer/issues/148

### CODE REVIEW

**Comment by joepavitt** [comment]
Thanks @Yndira-E - @gdziuba adding you in as a reviewer - this is the blog due for publishing today

**Comment by gdziuba** [comment]
reviewing now

File: src/blog/2024/03/dashboard-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+With all of this together, we have the following functional Dashboard:
+
+![Short animation showing the final functional dashboard](./images/dashboard-getting-started-final.gif){data-zoomable}
+<figcaption>Short animation showing the final functional dashboard.</figcaption>
+
+## Next Steps
+
+Whilst this is just a simple introduction of Node-RED Dashboard 2.0, we do have many other articles and documentation that can help you get started with more advanced features. 
+
+- [FlowFuse Dashboard Articles](https://flowfuse.com/blog/dashboard/) - Collection of examples and guides written by FlowFuse.
```
Comment (Hasmin-AC) [inline src/blog/2024/03/dashboard-getting-started.md:152]: This link doesn't take you to any dashboard articles.

File: src/blog/2024/03/dashboard-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+image: /blog/2024/03/images/getting-started-with-dashboard-2.png
+tags:
+    - posts
+    - releases
+    - community
+    - changelog
+    - dashboard
+---
+
+With our latest release of Node-RED Dashboard 2.0, we've made some big improvements to the onboarding experience.
```
[the remark this answers is not in the document]
Comment (joepavitt) [reply src/blog/2024/03/dashboard-getting-started.md:16]: Not accurate to link to that GA blog, as that was `v1.0.0`, this article refers to new functionality in `1.5.0+`. I could link to the GH Release Page: https://github.com/FlowFuse/node-red-dashboard/releases/tag/v1.5.0 but it's not very glamorous

[the remark this answers is not in the document]
Comment (gdziuba) [reply src/blog/2024/03/dashboard-getting-started.md:16]: This probably isn't the latest release, but if there is something we can link to that would be ideal.  Is there a change log equivalent?  Probably need to point out the version of the "latest" release.

File: src/blog/2024/03/dashboard-getting-started.md (hunk 1 above)
Comment (Hasmin-AC) [reply src/blog/2024/03/dashboard-getting-started.md:152]: ah my bad...it didn't look like dashboard articles at first :)

File: src/blog/2024/03/dashboard-getting-started.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+
+Whilst this is just a simple introduction of Node-RED Dashboard 2.0, we do have many other articles and documentation that can help you get started with more advanced features. 
+
+- [FlowFuse Dashboard Articles](https://flowfuse.com/blog/dashboard/) - Collection of examples and guides written by FlowFuse.
+- [Node-RED Dashboard 2.0 Documentation](https://dashboard.flowfuse.com) - Detailed information for each of the nodes available in Dashboard 2.0, as well as useful guides on building custom nodes and widgets of your own.
+- [Node-RED Forums - Dashboard 2.0](https://discourse.nodered.org/tag/dashboard-2) -  The Node-RED forums is a great place to ask questions, share your projects and get help from the community.
+- [Beginner Guide to a Professional Node-RED](https://flowfuse.com/ebooks/beginner-guide-to-a-professional-nodered/) - A free guide to an enterprise-ready Node-RED. Learn all about Node-RED history, securing your flows and dashboard data visualization.
+- [FlowFuse](https://flowfuse.com) - FlowFuse provides a complete platform to scale your production Node-RED applications, increase developer velocity, and enhance security in order to accelerate innovation.
```
[the remark this answers is not in the document]
Comment (joepavitt) [reply src/blog/2024/03/dashboard-getting-started.md:156]: I suspect a lot of people that read the Dashboard articles have no idea what FlowFuse is, I also suspect not many of them explore beyond the article as they don't realise the value offered - e.g. in the last 14 days,[ 56% of those reading a blog article have viewed another page on our website after that.](https://eu.posthog.com/project/2209/dashboard/2918?highlightInsightId=nQuGBogi)

This was my attempt at encouraging that - happy to change the URL

[the remark this answers is not in the document]
Comment (Hasmin-AC) [reply src/blog/2024/03/dashboard-getting-started.md:156]: Yeah I would suggest a more actionable link here like contact us than general flowfuse.

**Comment by joepavitt** [comment]
@gdziuba anything stopping this from being merged? Was due for publication today...

**Comment by gdziuba** [comment]
@joepavitt Nope.  I didn't see your response to Hasmin above.  It looked like an open dialog.  It can merge now.

**Comment by gdziuba** [comment]
@joepavitt I Will promote tomorrow.

**Pull request merged**